include/Model/Languages.php

```diff
@@ -414,20 +420,26 @@ public function delete( $lang_id ): bool {
 		}
 
 		// Delete menus locations.
-		if ( ! empty( $this->options['nav_menus'] ) ) {
-			foreach ( $this->options['nav_menus'] as $theme => $locations ) {
+		if ( ! empty( $this->options->get( 'nav_menus' ) ) ) {
+			$nav_menus = $this->options->get( 'nav_menus' );
```

Comment:
```suggestion
                 $nav_menus = $this->options->get( 'nav_menus' );
		if ( ! empty( $nav_menus ) ) {
```

Reply:
Done in [d78b566](https://github.com/polylang/polylang/pull/1595/commits/d78b566c2ac53d015116da9415cfd409770b70ff).